features/step_definitions/common_steps.rb

```diff
@@ -336,6 +336,24 @@
   end
 end
 
+Then /^I see an entry in the '(.*)' table with:$/ do |table_heading, table|
+  result_table_rows = get_table_rows_by_caption(table_heading)
+  expected_row = table.rows[0]
+  for row in result_table_rows
```

Comment:
You can also use 'each' loop, which is more idiomatic to Ruby, but then as we are a Python team, 'for' loop might be preferable for readability, your call really :).

https://stackoverflow.com/questions/3294509/for-vs-each-in-ruby
https://teamtreehouse.com/library/iteration-with-each